apps/bfd-server/bfd-server-launcher/src/test/java/gov/cms/bfd/server/launcher/AuthenticationIT.java

```diff
@@ -36,7 +36,7 @@ public void authenticationWorksForTrustedClient() throws IOException {
    *
    * @throws IOException (this exception indicates a test failure)
    */
-  @Test(expected = SSLHandshakeException.class)
+  @Test(expected = SSLException.class)
```

Comment:
Seems that newer versions of Java don't throw the explicit `SSLHandshakeException`.  The good news here is that `SSLHandshakeException` derives from `SSLException` so this should be true regardless of which is thrown.

Reply:
thats true